addons/web/static/src/core/emoji_picker/emoji_picker.js

```diff
@@ -77,6 +79,13 @@ export async function loadEmoji() {
                 }
                 loadingListeners.length = 0;
             }
+            loader.loaded.knownEmojisRegex = new RegExp(
+                Object.keys(loader.loaded.shortcodesByCodepoints)
+                    .map((c) => c.replace(/[-/\\^$*+?.()|[\]{}]/g, "\\$&"))
+                    .sort((a, b) => b.length - a.length) // Sort to get composed emojis first
+                    .join("|"),
+                "gu"
+            );
```

Comment:
Would remove "known" prefix, this is implicit in it being store in `emojiLoader.loaded`